pages/test_analytics/flaky_test_assignment.md

```diff
@@ -11,15 +11,21 @@ To enable assignments, you must have at least one team that has access to your s
 
 ## Assigning a test
 
-When viewing your flaky tests, you should now see a list of teams with suite access permissions listed inside the assignment dropdown. From here you may assign, reassign or remove the assignment.
+When viewing your flaky tests, you should now see a list of teams with suite access permissions listed inside the manage flaky dropdown. From here you may assign, reassign or remove the assignment.
 
 > 🚧 Assignment permissions
 > All team members have the ability to create, update or remove an assignment. This feature is not restricted to admins.
 
-Tests that are assigned to a team will be updated to display a badge indicating as such. This badge is also visible on issues in the run page.
+Tests that are assigned to a team will be updated to display a badge indicating as such.
 
 <%= image "flaky-test-teams.png", width: 1960/2, height: 630/2, alt: "Flaky test page showing team assignments" %>
 
+## Resolving a flaky test
+
+From the manage flaky dropdown, you can resolve your flaky test. Resolving a flaky test removes its current assignment and assigns a 'Marked as resolved' badge within the flaky index. If the test flakes again, it will be considered a 'Reoccurring flaky' and will receive a corresponding badge in the index.
```

Comment:
```suggestion
From the manage flaky dropdown, you can resolve your flaky test. Resolving a flaky test will remove the test from **My Assignments**. The flake will still appear on the flaky page, and display a **Marked as resolved** badge within the flaky index. If the test flakes again, it will be considered a **Reoccurring flaky** and will receive a corresponding badge in the index.
```

Also I think we're not meant to use quotes any more, you're meant to wrap these things in double asterisks

Reply:
I made another little update to the text to include a mention of it staying in the flaky index until it has not flaked within 7days.